### ISSUE TITLE
@supabase/auth-ui-react: if I use Github provider, A really big Github icon flashes on page load

### ISSUE THREAD

**Issue by Jared-Dahlke** [supabase/auth-ui#87]
# Bug report

## Describe the bug

A really big Github icon flashes on page load for only a few milliseconds, I took a video of it so i could find and take a screenshot of the correct frame, here it is:
![image](https://user-images.githubusercontent.com/32936978/208319274-063f50c4-f3c3-4271-81dc-a151e1c06bd4.png)

## To Reproduce

here is my login page:
```
export const Login: NextPage = () => {
  const session = useSession()
  const router = useRouter()
  const supabase = useSupabaseClient()

  return (
    <div className="mt-8">
      <Auth 
         providers={['github']} 
         supabaseClient={supabase} 
       />
    </div>
  )
}

```

## Expected behavior

I expect the login page to load with no flash of a github icon

## System information

 macbook
 chrome
 "@supabase/auth-ui-react": "^0.2.6",
 "@supabase/supabase-js": "^2.0.4",

**Comment by PhilBookst** [comment]
+ 1. Same behaviour here but with the google logo

**Comment by vymao** [comment]
Bump, same.

**Comment by fedorish** [comment]
If you're using Nextjs you can do step 1-3 [here](https://stitches.dev/blog/using-nextjs-with-stitches) since `getCssText` is exported from `@supabase/auth-ui-react` and the styles will be loaded on server. But it but it seems like it breaks the themes so not really a solution unless you use minimal/default theme.

**Comment by silentworks** [comment]
A fix was submitted by @rdylina that got released in version `0.3.5`.
